テーブル設計

AiAoHaru opened the pull request:
# What
- Furimaアプリケーションに必要なDBの設計

# Why
- 本アプリケーションのDBの概要を記載し、管理しやすくする為

AiAoHaru committed:
READMEに設計情報を記述

AiAoHaru committed:
READMEに設計情報を記述（書式修正）

AiAoHaru commented:
# **ER図**
![Furima ER図](https://user-images.githubusercontent.com/69007905/91332002-85c5c500-e806-11ea-8e88-7ff48e5e0458.png)

AiAoHaru commented:
# **ER図**
![Furima ER図](https://user-images.githubusercontent.com/69007905/91332305-f1a82d80-e806-11ea-9ab1-a507efa37ef6.png)

Katsuno0814 commented on README.md:
```diff
@@ -1,24 +1,106 @@
 
-* Database creation
+## profiles テーブル
 
-* Database initialization
+| Column              | Type       | Options                        |
+| ------------------- | ---------- | ------------------------------ |
+| family_name         | string     | null: false                    |
+| last_name           | string     | null: false                    |
+| family_name_reading | string     | null: false                    |
+| last_name_reading   | string     | null: false                    |
+| birth_date          | date       | null: false                    |
```
こちらのテーブルはuserにまとめましょう！

AiAoHaru replied:
レビューありがとうございます。
修正しました。

Katsuno0814 reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認をお願いいたします！

AiAoHaru committed:
READMEに設計情報を記述（修正：テーブル統合）

profiles テーブルをusers テーブルに統合

AiAoHaru commented:
# **ER図（修正後）**
![Furima ER図2（レビュー後）](https://user-images.githubusercontent.com/69007905/91377982-4b861300-e85a-11ea-8416-5ac9cb44d082.png)

Katsuno0814 commented on README.md:
```diff
@@ -4,33 +4,22 @@
 - has_many :items
 - has_many :comments, dependent::destroy
-- has_one :profile
 - has_one :purchaser
```
こちら
- has_many :purchasers
にしましょう！

AiAoHaru replied:
レビューありがとうございます。
ご指摘事項について、確認・修正しました。

一人のユーザーが複数の商品購入が可能なため、has_one:purchaser（単数形）ではなく、has_many :purchasers（複数形）という認識でよろしいでしょうか。

Katsuno0814 replied:
>一人のユーザーが複数の商品購入が可能なため、has_one:purchaser（単数形）ではなく、has_many :purchasers（複数形）という認識でよろしいでしょうか。

はい！こちら認識していただいた内容にて問題ありません！

Katsuno0814 reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認をお願いいたします！

AiAoHaru committed:
READMEに設計情報を記述（修正：アソシエーション）

users テーブルとpurchasers テーブルのアソシエーションを修正
前：has_one :purchaser
後：has_many :purchasers

AiAoHaru commented:
# **ER図（修正後2）**
![Furima ER図3（レビュー後）](https://user-images.githubusercontent.com/69007905/91380878-01ecf680-e861-11ea-8deb-9b1307b042c0.png)

Katsuno0814 reviewed (commented):
レビュー依頼ありがとうございます！
LGTMとさせていただきます！実装お疲れ様でした！
引き続き実装頑張っていきましょう！
